kusto/conn_streaming_ingest.go

```diff
@@ -0,0 +1,54 @@
+package kusto
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-kusto-go/kusto/data/errors"
+	"io"
+	"net/url"
+)
+
+type DataFormatForStreaming interface {
+	CamelCase() string
+	ForceDefault() DataFormatForStreaming
+}
+
+func (c *conn) StreamIngest(ctx context.Context, db, table string, payload io.Reader, format DataFormatForStreaming, mappingName string, clientRequestId string) error {
```

Comment:
Stream?

Reply:
Why? it's doing streaming ingestion. calling it stream will make you think it's a streaming query